### PR TITLE
executor, distsql: move "requestBuilder" from package "executor" to "distsql"

### PR DESCRIPTION
**Pull request by zz-jason**

### CODE REVIEW

File: distsql/request_builder.go [hunk 1]
```diff
@@ -0,0 +1,175 @@
+package distsql
```
Comment (shenli) [inline distsql/request_builder.go]: Miss licence.

**Comment by winkyao** [comment]
Plz fix ci. @zz-jason

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
@winkyao PTAL

**Pull request merged**